doc/release-notes.md

```diff
@@ -51,146 +55,215 @@ masternodes. This results in PoSe punishment/banning for outdated masternodes,
+  (or the `-bantime` setting). Depending on traffic from other peers,
+  discouragement may time out at an indeterminate time.
+
+- Discouragement is not persisted over restarts.
+
+- There is no method to list discouraged addresses. They are not returned by
+  the `listbanned` RPC. That RPC also no longer reports the `ban_reason`
+  field, as `"manually added"` is the only remaining option.
+
+- Discouragement cannot be removed with the `setban remove` RPC command.
+  If you need to remove a discouragement, you can remove all discouragements by
+  stop-starting your node.
 
 Remote Procedure Call (RPC) Changes
 -----------------------------------
 Most changes here were introduced through Bitcoin backports mostly related to
 the deprecation of wallet accounts in DashCore v0.17 and introduction of PSBT
 format.
 
-The new RPCs are:
-- `combinepsbt`
-- `converttopsbt`
-- `createpsbt`
-- `decodepsbt`
-- `deriveaddresses`
-- `finalizepsbt`
-- `getblockfilter`
-- `getdescriptorinfo`
-- `getnodeaddresses`
-- `getrpcinfo`
-- `joinpsbts`
-- `listwalletdir`
-- `quorum rotationinfo`
-- `scantxoutset`
-- `submitheader`
-- `testmempoolaccept`
-- `utxoupdatepsbt`
-- `walletcreatefundedpsbt`
-- `walletprocesspsbt`
-
-The removed RPCs are:
-- `estimatefee`
-- `getinfo`
-- `getreceivedbyaccount`
-- `keepass`
-- `listaccounts`
-- `listreceivedbyaccount`
-- `move`
-- `resendwallettransactions`
-- `sendfrom`
-- `signrawtransaction`
-
-Changes in existing RPCs introduced through bitcoin backports:
-- The `testnet` field in `dash-cli -getinfo` has been renamed to `chain` and
-now returns the current network name as defined in BIP70 (main, test, regtest).
-- Added `window_final_block_height` in `getchaintxstats`
-- Added `feerate_percentiles` object with feerates at the 10th, 25th, 50th,
-75th, and 90th percentile weight unit instead of `medianfeerate` in
-`getblockstats`
-- In `getmempoolancestors`, `getmempooldescendants`, `getmempoolentry` and
-`getrawmempool` RPCs, to be consistent with the returned value and other RPCs
-such as `getrawtransaction`, `vsize` has been added and `size` is now
-deprecated. `size` will only be returned if `dashd` is started with
-`-deprecatedrpc=size`.
-- Added `loaded` in mempool related RPCs indicates whether the mempool is fully
-loaded or not
-- Added `localservicesnames` in `getnetworkinfo` list the services the node
-offers to the network, in human-readable form (in addition to an already
-existing `localservices` hex string)
-- Added `hwm` in `getzmqnotifications`
-- `createwallet` can create blank, encrypted or watch-only wallets now.
-- Added `private_keys_enabled` in `getwalletinfo`
-- Added `solvable`, `desc`, `ischange` and `hdmasterfingerprint` in `getaddressinfo`
-- Added `desc` in `listunspent`
-
-Dash-specific changes in existing RPCs:
-- Added `quorumIndex` in `quorum getinfo` and `quorum memberof`
+### The new RPCs are:
+- The RPC `getrpcinfo` returns runtime details of the RPC server. At the moment
```

Comment:
`getrpcinfo` is not a new RPC. It was in v18.0. 
But, `dumptxoutset`, `generatetodescriptor`, `generateblock`, `mockscheduler` (hidden), and `sporkupdate` are all new.